src/components/Verification/interface.ts

```diff
@@ -60,6 +60,21 @@ export interface SchemaDetail {
 	schemaId: string;
 	credDefId: string;
 }
+export interface IW3cSchemaDetails {
+	schemaName: string;
+	version: string;
```

Comment:
Is `version` is required for W3C type schema

Reply:
Yes, It is required to display schema version on schema card.